lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
It doesn't seem like it: https://github.com/mperham/sidekiq/wiki/Scheduled-Jobs#checking-for-new-jobs